### PR TITLE
Extract common indent constant

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

File: tools/licenses/lib/patterns.dart [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 // COMMON PATTERNS
 
-// TODO(ianh): factor out the indent pattern which is duplicated everywhere
+const String kIndent = r'^((?:[-;#<!.\\"/* ]*(?:REM[-;#<!.\\"/* ]*)?[-;#<!.\"/*]+)?)( *)';
```
Comment (eseidelGoogle) [inline tools/licenses/lib/patterns.dart]: Effective Dart disagrees with your hungarian naming. :D  (Although I personally agree.)

Comment (Hixie) [inline tools/licenses/lib/patterns.dart]: Hungarian naming would be sIndent!

This is Flutter Style Guide naming. :-P

**Comment by Hixie** [comment]
LGTM

**Pull request merged**